src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolver.cs

```diff
@@ -215,7 +215,10 @@ private static bool IsRenameValid(MutableConflictResolution conflictResolution,
             try
             {
                 var projectOpt = conflictResolution.CurrentSolution.GetProject(renamedSymbol.ContainingAssembly, cancellationToken);
-                if (renamedSymbol.ContainingSymbol.IsKind(SymbolKind.NamedType))
+
+                // For tuple fields the ContainingAssembly is System.ValueTuple, so we can't get a project
+                // but fortunately we know there are no methods or parameterized properties either
+                if (renamedSymbol.ContainingSymbol.IsKind(SymbolKind.NamedType) && !renamedSymbol.IsTupleField())
```

Comment:
Why the NamedType check? IsTupleField will catch that just fine, right?

Reply:
The named type check is pre-existing.